go.mod

```diff
@@ -34,29 +34,30 @@ require (
 	github.com/digitalocean/godo v1.124.0
 	github.com/distribution/distribution/v3 v3.0.0-beta.1
 	github.com/distribution/reference v0.6.0
-	github.com/envoyproxy/go-control-plane v0.12.1-0.20240621013728-1eb8caab5155
+	github.com/envoyproxy/go-control-plane v0.13.0
 	github.com/equinix/equinix-sdk-go v0.46.0
-	github.com/go-git/go-git/v5 v5.11.0
+	github.com/go-git/go-git/v5 v5.12.0
 	github.com/go-logr/zapr v1.3.0
 	github.com/go-test/deep v1.1.1
 	github.com/gobuffalo/flect v1.0.2
 	github.com/google/go-cmp v0.6.0
-	github.com/google/go-containerregistry v0.19.1
+	github.com/google/go-containerregistry v0.20.2
 	github.com/google/uuid v1.6.0
 	github.com/gophercloud/gophercloud v1.14.0
 	github.com/hetznercloud/hcloud-go v1.57.0
 	github.com/jackpal/gateway v1.0.14
 	github.com/kubermatic/grafanasdk v0.9.13
+	github.com/kyverno/kyverno v1.13.2
```

Comment:
Are all those additional ton of dependencies come just from this import?

Reply:
They do, we discussed copying the neccessary API types from Kyverno (in our case the Policy Spec), but at the moment it was decided to leave it as is because there will be other parts that use it on seed integration and controllers (outside this pr).